.github/workflows/container-image.yml

```diff
@@ -182,3 +176,53 @@ jobs:
             ghcr.io/${{ github.repository_owner }}/policy-server@${{ steps.build-latest.outputs.digest }}
         env:
           COSIGN_EXPERIMENTAL: 1
+
+      - name: Install Golang
+        uses: actions/setup-go@v3
+        with:
+          go-version: '1.17'
+
+      - name: Install the bom command and prepare destination directory
+        shell: bash
+        run: |
+          set -e
+          go install sigs.k8s.io/bom/cmd/bom@v0.2.2
+          mkdir policy-server-container-image-sbom
```

Comment:
Please, create a dedicated github actions (like we do with kwctl) to install the `sbom` tool. Also, do not build it from source but use a released version. [This](https://github.com/kubernetes-sigs/bom/releases/tag/v0.3.0)  is the latest one.

Reply:
Done